axonivy/update.axonivy.com
# Update dependencies

- Fix writing errors on local dev system in Dockerfile
- Update composer to latest
- Update all composer dependencies to latest
- Fix PHPUnit deprecation warning
- Add more log output if there are warnings in phpunit tests
- Use port 8080 for local development because docker requires now root when using port 80
- No longer lock php dependencies to versions in composer.json (they are locked in composer.lock)
